src/components/attendee/PortalModal/PortalModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { MouseEventHandler } from "react";
+import { Popover, PopoverProps } from "components/attendee/Popover";
+
+import { Room } from "types/rooms";
+
+import { useSpaceById } from "hooks/spaces/useSpaceById";
+
+import { Button } from "../Button";
+
+import CN from "./PortalModal.module.scss";
+
+interface PortalModalProps extends PopoverProps {
+  onEnter?: MouseEventHandler<HTMLButtonElement>;
+  portal: Room;
+}
+
+export const PortalModal: React.FC<PortalModalProps> = ({
+  referenceElement,
+  onEnter,
+  portal,
+}) => {
+  const { space } = useSpaceById({ spaceId: portal.spaceId });
```

Comment:
Or, maybe if TS doesn't complain, just

```suggestion
  const { space } = useSpaceById( portal );
```

Otherwise it's a nice to have polish we can do later on.

Reply:
ts is complaining.. although i can't really get why. spaceId is optional in both cases.
<img width="852" alt="image" src="https://user-images.githubusercontent.com/63313732/160836045-010ebbde-96d5-4c86-967f-e5e5592dc82e.png">

anyway, left it as is.